src/_vendor/zod-to-json-schema/parsers/optional.ts

```diff
@@ -3,6 +3,15 @@ import { JsonSchema7Type, parseDef } from '../parseDef';
 import { Refs } from '../Refs';
 
 export const parseOptionalDef = (def: ZodOptionalDef, refs: Refs): JsonSchema7Type | undefined => {
+  if (refs.openaiStrictMode) {
+    const fieldName = refs.propertyPath?.slice(-1)[0] || 'unknown';
```

Comment:
Since refs.propertyPath is an array, can we replace `.slice(-1)[0]` with `.at(-1)` here? Using .at(-1) improves readability and performance by directly accessing the last element without creating a new array.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at